parser-typechecker/src/Unison/Codebase/Editor/HandleInput.hs

```diff
@@ -695,19 +698,16 @@ loop = do
           Right selection -> do
             let allMd' = Map.restrictKeys allMd selection
                 allRefs = toList (Set.unions (Map.elems allMd'))
-            termDisplays <- Map.fromList <$> do
-              terms <- filterM (eval . IsTerm) allRefs
-              traverse (\r -> (r,) <$> loadTermDisplayThing r) terms
-            --   typeDisplays <- Map.fromList <$> do
-            --     types <- filterM (eval . IsType) allRefs
-            --     traverse (\r -> (r,) <$> loadTypeDisplayThing r) types
-            ppe <- prettyPrintEnv =<< makePrintNamesFromLabeled' (deps termDisplays)
-            respond $ DisplayLinks ppe allMd' mempty termDisplays
-          where
-            deps :: Map Reference (DisplayThing (Term v Ann)) -> Set LabeledDependency
-            deps dts = Set.map LD.termRef (Map.keysSet dts)
-                    <> foldMap Term.labeledDependencies
-                               (mapMaybe DT.toMaybe $ Map.elems dts)
+                results :: Set Reference = Set.unions $ Map.elems allMd'
```

Comment:
Note: simplified logic here - `links` command just shows a menu of results, doesn't print them out. You use `display` or `view` to look at the metadata.